lib/core/event_handler.ex

```diff
@@ -31,9 +31,9 @@ defmodule TelemetryMetricsPrometheus.Core.EventHandler do
   @spec get_measurement(:telemetry.event_measurements(), atom()) ::
           {:ok, number()} | measurement_not_found_error() | measurement_parse_error()
   def get_measurement(measurements, measurement) when is_atom(measurement) do
```

Comment:
Should we also cover the case when measurement function returns a bad value?

Reply:
Yeah. That's a simple add.

Reply:
Good call